packages/framework/src/Component/Elasticsearch/IndexFacade.php

```diff
@@ -167,7 +168,15 @@ public function migrate(IndexDefinition $indexDefinition, OutputInterface $outpu
     {
         $indexName = $indexDefinition->getIndexName();
         $domainId = $indexDefinition->getDomainId();
-        $existingIndexName = $this->indexRepository->findCurrentIndexNameForAlias($indexDefinition->getIndexAlias());
+
+        try {
+            $existingIndexName = $this->indexRepository->findCurrentIndexNameForAlias($indexDefinition->getIndexAlias());
+        } catch (ElasticsearchIndexException $exception) {
+            if (strpos($exception->getMessage(), 'Can\'t found any index with alias') === false) {
```

Comment:
This is quite fragile, especially when I see the exception message that will be probably proposed for fixing sooner or later - "Can't found" is wrong, should be rather "Not able to find" :grin: 

What about throwing and catching more specific Exception? :slightly_smiling_face: